pkg/bot-code/client.go

```diff
@@ -0,0 +1,290 @@
+// client.go
```

Comment:
refactor: 
- All of these changes should be applied to the already existing file `pkg/bot-github/client.go` instead
- afterwards please remove this file `pkg/bot-code/client.go`